src/Controls/src/Core/EnumerableExtensions.cs

```diff
@@ -60,5 +59,25 @@ static class EnumerableExtensions
 				}
 			}
 		}
+
+		public static bool HasAnyGesturesFor<T>(this IEnumerable<IGestureRecognizer>? gestures, Func<T, bool>? predicate = null) where T : GestureRecognizer
+		{
+			if (gestures is null)
+			{
+				return false;
+			}
+
+			predicate ??= x => true;
```

Comment:
Could we just do a null check inside the `foreach`?
```csharp
if (item is T gesture && (predicate is null || predicate(gesture)))
{
	return true;
}
```
This way we aren't creating a lambda like `x => true`.